tests/e2e/ctl_v2_test.go

```diff
@@ -229,6 +230,9 @@ func TestUtlCtlV2Backup(t *testing.T) {
 func testUtlCtlV2Backup(t *testing.T, snapCount int, v3 bool, utl bool) {
 	BeforeTestV2(t)
 
+	revertFunc := verify.DisableVerification()
```

Comment:
Can you provide context why we disable verification here?

Reply:
sure, it's because the case offline updated the bbolt db file. Will add a comment.